make/harbor.yml

```diff
@@ -139,3 +133,15 @@ _version: 1.8.0
 # Uncomment uaa for trusting the certificate of uaa instance that is hosted via self-signed cert.
 # uaa:
 #   ca_file: /path/to/ca
+
+# Global proxy
+# Config http proxy for components, e.g. http://my.proxy.com:3128
+# components doesn't need to connect to each others via http proxy.
+proxy:
+  http_proxy:
+  https_proxy:
+  no_proxy: 127.0.0.1,localhost,.local,.internal,core,jobservice
```

Comment:
Add other components to the no_proxy list and add more comments to explain how to use these configurations